src/constants.js

```diff
@@ -42,12 +42,22 @@ module.exports = {
      * The interval in which to batch and send events.
      */
     EVENTS_PING_INTERVAL: 60 * 1000, // 5s
-    
+
     /**
      * The current version of this SDK,
      * this must by in sync with the version in the package.json.
      *
      * TODO: Find a better way to get this.
      */
     TELEMETRY_SDK_BUILD: "0.0.1",
+
+    /**
+     * The telemetry endpoint to send data to.
+     */
+    TELEMETRY_ENDPOINT: "https://incoming.telemetry.mozilla.org/",
```

Comment:
Are we stubbing this in tests?

Reply:
Not yet.

Reply:
I stand correctly, yesterday me actually stubbed the fetch function. See lines 16 through 20 of the tests.